bind/gencffi_cdef.go

```diff
@@ -68,6 +122,52 @@ func (g *cffiGen) genCdefFunc(f Func) {
 	g.wrapper.Printf("extern %[1]s cgo_func_%[2]s(%[3]s);\n", cdef_ret, f.id, paramString)
 }
 
+// Generate a C definition of Go type method.
+func (g *cffiGen) _genCdefFunc(sym *symbol, fsym *symbol) {
```

Comment:
the name of this method isn't too Go-idiomatic (looks more like python :))
for symmetry with `genTypeCdefInit`, I would perhaps call it `genTypeCdefFunc`, or `genTypeCdefMeth`.

we don't like underscores very much :)